package.json

```diff
@@ -45,6 +45,7 @@
     "core-js": "^3.6.5",
     "cssnano": "^4.1.10",
     "dom-autoscroller": "^2.3.4",
+    "drawing-tool": "^2.0.3",
```

Comment:
Remove? It looks like this dependency is not being used. The code is referencing lib/drawingTool/drawing-tool.js